lib/galaxy/config.py

```diff
@@ -1054,6 +1056,9 @@ def _configure_toolbox(self):
             involucro_auto_init=self.config.involucro_auto_init,
         )
         self.container_finder = containers.ContainerFinder(app_info)
+        self.toolbox.dependency_manager.resolver_classes.update(self.container_finder.container_registry.resolver_classes)
+        self.toolbox.dependency_manager.dependency_resolvers.extend(self.container_finder.container_registry.container_resolvers)
```

Comment:
This will append enabled container resolvers to the regular dependency resolvers. Do we think this is good enough or do we want finer-grained control, and if yes, should we maybe merge the container_resolvers_conf.xml file into the dependency_resolver_conf.xml file ?

Of note the regular dependency resolver will pick a dependency, but the dependency resolver runs after that and might overwrite the chosen dependency (this is pre-existing, but I guess that would be also up to debate ?).

I think for my own needs this is fine as is, so let me know.